server/app/views/applicant/ApplicantCommonIntakeUpsellTemplate.html

```diff
@@ -0,0 +1,57 @@
+<html th:attr="lang=${preferredLanguage.code()}">
+  <head
+    th:replace="~{applicant/ApplicantBaseFragment :: pageHeaderScriptsAndLinks}"
+  ></head>
+  <body data-load-modal="true">
+    <div th:replace="~{applicant/NavigationFragment :: pageHeader}"></div>
+    <main role="main">
+      <h1
+        th:text="#{title.commonIntakeConfirmationTi}"
+        th:if="${isTrustedIntermediary}"
+      ></h1>
+      <h1
+        th:text="#{title.commonIntakeConfirmation}"
+        th:if="${!isTrustedIntermediary}"
+      ></h1>
+
+      <div th:if="${eligiblePrograms.size()==0}" style="margin-bottom: 20px">
```

Comment:
use a uswds style here instead https://designsystem.digital.gov/utilities/margin-and-padding/ instead of specifying style directly

I think you want class="margin-bottom-205"

Reply:
same thing elsewhere in this file